tensorflow_probability/examples/jupyter_notebooks/Fitting_DPMM_with_Symmetric_Dirichlet_Prior_Using_pSGLD.ipynb

```diff
@@ -0,0 +1,975 @@
ogJRPiF79m17pX5jN336sn\nMHkGW7lOngElbaTadpWENdOZIt4PJU3D77YWAHJvXvNheqt97IbcDju6PuzG/S5B68imlSW1mi9t\nJodqnpS1XtsL51EPPiGNnqFJAuljYbp1iwpi8xay7npIhp7LfcBKWVBrAGO9kCgtpodqB0V8rGFs\nbJP3ObBzi/5ccWCqZNE8Rkq0HvIf/YPCNb6BakkigMmPQaRmUu/9F79WW8RqeXATAAlcKINpwTxI\nKZvmqq3vldJilkXaIss1t48ddtjRpfGjWr7+1OiO3X9kSRG7sc1bSAfZEFY1mUwsiVJ7bWsGHDCw\n4w3HMS3MNYfcpQRweB+Nxsj7AUh6f0LQwGSM5fOROcC2L8iWv/8R5uHLTrF/+aXzQE9H4KaqD9/L\n0XbTFTssERHD3DpAzz3vc31bcjqwbycwcjwV5yY8AmzaQGW7/d8yBP/AdGBQkplRbhR5kXvzVA2D\nxRBCvUcW5ULMX/z/2jvz+CjLq+9/r0lARAkkQMgOCWEn7KsQUJFVXEpFBKugtc/bx1a0PspW29qP\ntRKoVgSe+lqtxIqA+lpBxAUBWVQEF2QVZc0CBAQisolkrvePc8+WTBZCIJnhfD8fPjO5577nvq+5\nZzjXOdc5v4OraXrAZRQP1ZdVo17aMSCdqfLz88s9NlS5FDqLhev4wnlsUDVd4dRzry6S04j92wti\nrC8x0VgAACAASURBVIvLiBYrVfJmNBtKlEC53RbG/hab1MwJp6ZB+y7ija98R/516QMY6HOdz9Cv\nXgpYWc9dPB/y98ibfue0v/zJz5j7G/bYYl+6+JQq+0hClgaN5PGM0winWQsYMVb09zv3BozkM/Qf\nCr2vlYjIonny+NU6uPsB6DcY3nwZtnwJubtLet7ZM0X9LSVNytEK8mHitEAxGIfiYfWKhNlL3Uc9\nekUJSdRzvwiU5jmVNvusqOdlc3bifuJhPE01bPZM8QjfflUe33xZkrM2rvdluDulcrRsJy1HPYpq\nHoyBjr0A63SFK46hTOlWRYhPltp2kLX2Ja/KEslNt8PiedIm1gBxyXKPp04Qz3/7Rpg8HVdkZKn3\n39vT/J7/Oe+e5mWhnntoE87jC+exgXruocM5ej/+XpQv0SnIjslpEqadNE1qpK+/VdZzr79Vaqpv\nul3C7cNu9R2TOUi8+u2bJKErNk62t+og6nPWwoZPID9HtkfWkscmic4bqGEPSnHhm9SWvuf5eyVC\n0rmX6AfcMBqaJIjWAFaiLcNH+brwbf2qbE+7u5OA1z3zggzFH02uU5TQRI37xeA8BDpszi7sExPE\nsys2OTBGdMC9DUkWL5BWom/Nh8/XQKMmYsAzusIvfyfiNKveAyzc9YAYmQ2fikTtN5ulD7uHQ86s\n2OPx++vHKyU5e8bRkkekeD9ZLs8794YrrpR8hy/XwtmzsPAVyaAvKoL9ebg/XQnRjfD2lW/bUaIy\nbnfQjHiXyyUqha7gP9/yKisURQl/1LjXeBypuXGlZDp7JEsL9uGaPE0yrO+6H+bMlDK4ceNFqrbn\n1ZjJf5PyqoVzpR3r4gWQORC2bZRJQapT11znSnls3010v5XgxCcH/t3UWf8+fMBXaXD8B0mgy+gG\nWPHejZFJlsslanQvPAUvzZJufL9/EhMRiXvqROz6VbifeBi7L0cSJsuZHPoqJ6punVwnCooSmqhx\nvxicR1KSVxfeqXMuju3WV+qmu/X1itnQPVNC89ZCbLz8Z793B9btFm8RIx7k8FE+T37RK77e5Kcd\n0ZrNn/mEbJSSeNbUPWz8rOQ+3zplhUedMrkvP5GJ1I2jYdz9Ivn7ywdh7H3w1nyMcUmS5cQsmdcV\nuX3h+zJC4wG17AQpj6ssxb67auwVJTTQlq8Xg/MIy5fV3tJai/lsDfbtV3FldPW1+szbjV08XzzE\ngn2SaOdpSVrktDtdPF+S61q2l/V3kKQvDx0cCVUNx1ccz2ccjJwd8niDoymwcK6E6uOTcCUk4z5b\nBEVnsRvXS2tXA/alWRJpWbwAb8je/3R+JZJgpVRu3PiqXSMv/t0N0lpWUZSah2rLXwRK082uiD6y\nN1M6qmQrTJuzCzvzMRg3HtO2U4CuuI2Nk/awQ36OyRwEV10LGGntOnw0xCfCnh0+b334bZLk5Wn9\nWpAPJ36QGndPu1el8iSnw7EjEJ8Ey96SOvicXdCiHXb24xDdUFQBt2+CyEhsl6ukZW+/Id6eAUDg\ndyF3l+RirPkA03cQrsyBgd+D86RevXocP3488Lt7Dq1lazqXgj55uI4vnMcGVaMtr557TacUT8la\ni92fC8Zp2YmUxnk8LBOXhJ2YJbXxKWmwbhV21bvirTeOg8ZNpNbd7YYbb5c1+NXvyev+HqiK15SP\nKxLczgQoNlE6xXkxEkHJ3SF6/avfl5r2Uffgyhwomv/xSbgTm0HDxiJHu2geAHbxArmPWOzenZI4\n+dIs33fBqZaQCLmF5Auf1V5WJElRlJqDGveaTmkh/dxdshZ7w23e1p2iVDcVDuRj5zwj/bcXzcPe\nOBoG/Qy6fCK65S8+Lcp0bseIL57ne158LfXK+nD8+ws/zlDG7RfZ8Bj2G0bLOnu602Bn93ZY9b58\nvh8uAcCO/i9vuaPJ24PpeTXus2dF/nfhK3DTGKy7CLImAkaOvet+p52vowHfNN2nUngeofKKqNgp\nihI6aEJdDafUOuPkNMmKf/tVyN0tBmJiFmBkjX34KFmr9SifzX9eyt76DYW77se3hutn5AE6dvc9\nb5JUvmGPuATnh1dE+ZrqJDULvs+SV6W1679nySRstWPYO/eWCoUPl2DXrZREx78+hH3iYTGur74A\nX38l0ZO35sHB/YCR+2nkqc0qluBmkUnd+STQqRKdooQVl+D/zKGHx6uySamBmuPdMyWVyrph2mRc\nk7JEmnTsfRJ6
```

Comment:
Maybe I am missing this (since it is not rendering), but it would be nice to say what the result of pSGLD is (i.e. distinguish between a traditional optimizer, vs.  one which is a MC scheme to generate posterior samples).

Reply:
Added the additional result on fitting the model using RMSProp as traditional optimizer. I mentioned the processing time, visualized the clustering result and mixture weights of the components for easy comparison.
I also added the experiment on uncertainty to distinguish pSGLD from MAP inference of traditional optimizer.